CHANGELOG.md

```diff
@@ -12,6 +12,39 @@ Bugfixes:
 
 Other improvements:
 
+## [v5.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v4.1.0) - 2021-MONTH-DATE
+
+Breaking changes:
+- Make library compile on `v0.14.0` (#206)
```

Comment:
I would suggest phrasing this as "Support compiler version `v0.14.0`, and drop support for previous versions", as it's a bit more clear that `v0.14.0` refers to the compiler version that way as opposed to the version of some other tool, and also it means that it's clear that this version of the library can't be used with older compilers.